src/SFA.DAS.ProviderRegistrations/Application/Commands/SignedAgreementCommand/SignedAgreementCommandHandler.cs

```diff
@@ -23,6 +23,10 @@ protected override async Task Handle(SignedAgreementCommand request, Cancellatio
             {
                 var invitation = await _db.Value.Invitations.SingleOrDefaultAsync(i => i.Reference == Guid.Parse(request.CorrelationId) && i.Status < (int) InvitationStatus.LegalAgreementSigned, cancellationToken);
                 invitation?.UpdateStatus((int) InvitationStatus.LegalAgreementSigned, DateTime.Now);
+
+                var invitationEvents = new InvitationEvent(invitation?.Id, (int)EventType.LegalAgreementSigned, DateTime.UtcNow);
```

Comment:
I think we should be passing down the Datetime from the NServicebus event properties